jest.config.js

```diff
@@ -2,12 +2,13 @@ module.exports = {
   transform: {
     '^.+\\.tsx?$': 'babel-jest',
   },
+  transformIgnorePatterns: [],
   moduleNameMapper: {
     '\\.(css)$': 'identity-obj-proxy',
     '\\.(scss)$': 'identity-obj-proxy',
     '^@carbon/icons-react/es/(.*)$': '@carbon/icons-react/lib/$1',
     '^carbon-components-react/es/(.*)$': 'carbon-components-react/lib/$1',
-    '@openmrs/esm-framework': '<rootDir>/__mocks__/openmrs-esm-framework.mock.tsx',
+    '@openmrs/esm-framework': '@openmrs/esm-framework/mock.tsx',
```

Comment:
I think this can also drop the `.tsx` suffix (at least it was used so in the other repo, and we should use it consistently)

Reply:
Resolved